tools/osbuilder/Makefile

```diff
@@ -100,7 +100,7 @@ $(ROOTFS_BUILD_DEST)/.%$(ROOTFS_MARKER_SUFFIX):: rootfs-builder/%
 .PRECIOUS: $(ROOTFS_BUILD_DEST)/.dracut$(ROOTFS_MARKER_SUFFIX)
 $(ROOTFS_BUILD_DEST)/.dracut$(ROOTFS_MARKER_SUFFIX): $(TARGET_INITRD)
 	mkdir -p $(TARGET_ROOTFS)
-	(cd $(TARGET_ROOTFS); cat $< | cpio --extract --preserve-modification-time --make-directories)
+	(cd $(TARGET_ROOTFS); zcat $< | cpio --extract --preserve-modification-time --make-directories)
```

Comment:
The change itself makes sense, and just for completeness, here's where the initrd is gzipped: https://github.com/kata-containers/kata-containers/blob/8ae14f6a5565b7e759be9b9e0643c818e599bf4d/tools/osbuilder/initrd-builder/initrd_builder.sh#L79

Some of the checks in our CI are going to complain that there's no `Fixes: #NNN` in your commit message, as the community requires an issue to be created and linked to the PR's commit message.

Last but not least, I wonder why the CI is not hitting this case at all.

Reply:
I looked into this, and there is some confusing with the makefile targets:
* the default target should be `all`, not `$(ROOTFS_BUILD_DEST)/.dracut$(ROOTFS_MARKER_SUFFIX)`
* `$(ROOTFS_BUILD_DEST)/.dracut$(ROOTFS_MARKER_SUFFIX)` only be buildable when `BUILD_METHOD=dracut` is set.

So my fix is wrong, for the dracut case the initrd is actually not compressed. I'll open a new PR.